plugin/trino-elasticsearch/src/main/java/io/trino/plugin/elasticsearch/ElasticsearchTableHandle.java

```diff
@@ -32,12 +36,14 @@ public record ElasticsearchTableHandle(
         TupleDomain<ColumnHandle> constraint,
         Map<String, String> regexes,
         Optional<String> query,
-        OptionalLong limit)
+        List<TermAggregation> termAggregations,
+        List<MetricAggregation> metricAggregations,
+        Optional<TopN> topN)
```

Comment:
Instead of maintaining `TopN` which maintains `limit` as well - Can we have an `Optional<List<SortOrder>>` if the list is present we could consider it as `TopN` else `Limit` - But we need to handle what if `TopN` -> `Limit` and otherway around

Reply:
@Praveen2112 , can we keep the topN instead of splitting two arguments Optional<List<SortOrder>> and limit? But if that is what we need to do, we can look into changing accordingly.